db/citations.yml

```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
   volume: '88'
   wwwemail: ''
   year: '2013'
+thatc;arxiv21:
+  author:
+    - Zachary Thatcher
+    - Chia-Hao Liu
+    - Long Yang
+    - Brennan C. McBride
+    - Gia Thinh Tran
+    - Allison Wustrow
+    - Martin A. Karlsen
+    - James R. Neilson
+    - Dorthe B. Ravnsbæk
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: nslsii, xpd, desy, P02.1
+  grant: efrc18
+  journal: arXiv:2112.13300 [cond-mat]
+  month: dec
+  nb: ''
+  note: ''
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: '2112.13300'
+  title: 'nmfMapping: a cloud-based web application for non-negative matrix factorization
```

Comment:
this will have to be `{nmfMapping}` to compile correctly in latex

Reply:
resolved